hera/GameMap.tsx

```diff
@@ -1721,12 +1722,26 @@ export default class GameMap extends Component<Props, State> {
             {(propsShowCursor || propsShowCursor == null) &&
               showCursor &&
               !replayState.isReplaying && (
-                <Cursor
-                  position={position}
-                  size={tileSize}
-                  zIndex={zIndex - 4}
-                />
+                <>
+                  <Cursor
+                    position={position}
+                    size={tileSize}
+                    zIndex={zIndex - 4}
+                  />
+                  {editor?.mode === 'design' && (
+                    <MapEditorExtraCursors
+                      // TODO: adjust color
+                      color="red"
```

Comment:
@cpojer PTAL

Reply:
I think I'm just gong to leave them all red for now. It's easier to parse. Might change it in the future.